### PR TITLE
test ,what to do?

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: config/passport.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  passport.use('local-signup', new LocalStrategy({
+
+    usernameField: 'email',
+    passwordField: 'password',
+    passReqToCallback: true,
+  },
+  (req, email, password, done) => {
+    process.nextTick(() => {
+      User.findOne({ email }, (err, user) => {
```
Comment (InnaKutsik) [inline config/passport.js]: move to backendQuery folders in order to reuse the same methods

File: controllers/register_controller.js [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+  return encrypt.hashPassword(password)
+    .then((result) => {
+      const newUser = new User({
+        _id: new mongoose.Types.ObjectId().toHexString(),
+        first_name: firstName,
+        last_name: lastName,
+        email,
+        password: result,
+      });
+      return newUser.save();
```
Comment (InnaKutsik) [inline controllers/register_controller.js]: move to backendQuery folders in order to reuse the same methods

Comment (roman454647) [reply controllers/register_controller.js]: done

File: config/passport.js (hunk 1 above)
Comment (roman454647) [reply config/passport.js]: done

File: controllers/register_controller.js [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
-    }
-    const {
-        first_name: firstName, last_name: lastName, email, password,
-    } = req.body;
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(422).json({ errors: errors.array() });
+  }
+  const { email, password } = req.body;
+  return User.findOne({ email: req.body.email })
```
Comment (InnaKutsik) [inline controllers/register_controller.js]: MOVE TO USER QUERY TO DB

File: db/query/findById.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const User = require('../models/User');
+
+exports.findUser = (id, done) => {
```
Comment (InnaKutsik) [inline db/query/findById.js]: CREATE FILE WITH USERqUERY

AND ALL QUESRY RELATED TO uSER SCHEME MOVE THERE

Comment (InnaKutsik) [inline db/query/findById.js]: FindUserById

Comment (roman454647) [reply db/query/findById.js]: done

Comment (roman454647) [reply db/query/findById.js]: created

File: controllers/register_controller.js (hunk 3 above)
Comment (roman454647) [reply controllers/register_controller.js]: done

**Pull request merged**